CMakeLists.txt

```diff
@@ -0,0 +1,234 @@
+cmake_minimum_required (VERSION 2.8.11)
+project (libpd C)
+
+option(BUILD_SHARED_LIBS "Build libpd as a shared library instead of static" ON)
```

Comment:
Why not both directly?

Reply:
By the way, the static has been totally removed from my CMakeList

Reply:
I agree. Building both makes sense for Windows targets.

Reply:
BUILD_SHARED_LIBS is a standard toggle flag in cmake. if you set it to OFF, it will build a static library.
We can also build both at the same time, but static and dynamic libraries share a .lib file, which means you need to have a different file name.
Normally this is solved by generating something like pd.dll, pd.lib (for dynamic libs) and pd-static.lib
Would you like that?

Reply:
Perhaps more libpd.dll/libpd.lib & libpd-static.lib?  But libpd-static should be only for Windows, Linux and MacOS use different extensions for dynamic and static libraries. Another solution would be to use different output folders.

Reply:
"libpd.dll" as the Pure Data desktop app builds a "pd.dll" Also "libpd-static.dll/lib" sounds good to me.